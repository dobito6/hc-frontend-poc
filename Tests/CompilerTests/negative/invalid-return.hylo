struct A {}

fun f0() {
  ((),)
}

fun f1() {
  return ((),)
}

fun f2() -> A {
  return
}
