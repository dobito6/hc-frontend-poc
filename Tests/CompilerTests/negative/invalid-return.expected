negative/invalid-return.hylo:4.3-8: error: no coercion from '{Void}' to 'Void'
  ((),)
  ~~~~~
negative/invalid-return.hylo:8.10-15: error: no coercion from '{Void}' to 'Void'
  return ((),)
         ~~~~~
negative/invalid-return.hylo:12.3: error: expected value of type 'A'
  return
  ^
